sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTest.java

```diff
@@ -720,18 +706,14 @@ public void listCertificateIssuers(HttpClient httpClient, CertificateServiceVers
             for (CertificateIssuer issuer : certificateIssuersToList.values()) {
                 CertificateIssuer certificateIssuer = certificateClient.createIssuer(issuer);
 
-                assertTrue(issuerCreatedCorrectly(issuer, certificateIssuer));
+                assertIssuerCreatedCorrectly(issuer, certificateIssuer);
             }
 
             for (IssuerProperties issuerProperties : certificateClient.listPropertiesOfIssuers()) {
                 certificateIssuersToList.remove(issuerProperties.getName());
             }
 
             assertEquals(0, certificateIssuersToList.size());
-
-            for (CertificateIssuer issuer : certificateIssuers.values()) {
```

Comment:
I think so, I don't fully remember why this was done. Overall, it seemed tests were hit or miss on cleaning up their resources and I thought it would be better to just remove this.